randomkick.py

```diff
@@ -135,6 +135,17 @@ async def kick_user(delay, *, warn):
                 await asyncio.sleep(random.randint(7, 10))
             await bot.send_message(GROUP, 'Oh darn! That was close 😅')
 
+        warn_message = await bot.get_message(GROUP, event.id)
+        # message deleted or expired
+        if not warn_message:
+            await bot.send_message(
+                GROUP,
+                'Who the he*ck deleted my warning message? 3:<\n'
+                f'Guess I shouldn\'t kick <a href="tg://user?id={chosen.id}">{chosen.name}</a>'
+                ' for being inactive…',
+                parse_mode='html')
+            # clear wait delay
+            chosen.clicked_save()
```

Comment:
This needs a `return` (or `else` for the rest of the code)

```suggestion
            chosen.clicked_save()
            return
```